The apache mod_jk modules *communicate* with Java using an internet
protocol, AJP13, but they are written in C.  I can't come up with any
reason to look for a JDK here, it's plain stupid! Since apache and
tomcat can be run on separate machines, there is no dependency
whatsoever... /Palle

--- configure.in.orig	Tue Oct 22 00:00:00 2002
+++ configure.in	Thu Feb 26 00:19:35 2004
@@ -184,10 +184,10 @@
 
 dnl Java settings
 
-JK_JDK()
-JK_JDK_OS()
-JK_JNI()
-JK_PCRE()
+dnl JK_JDK()
+dnl JK_JDK_OS()
+dnl JK_JNI()
+dnl JK_PCRE()
 
 AC_SUBST(JAVA_HOME)
 AC_SUBST(JAVA_PLATFORM)
